schema.sql

```diff
@@ -4,14 +4,35 @@ CREATE DATABASE vet_clinic;
 
 \c vet_clinic;
 
-CREATE TABLE IF NOT EXISTS animals(
-  id              int GENERATED ALWAYS AS IDENTITY, 
-  name            varchar(100) not null,
-  date_of_birth   date not null,
-  escape_attempts int not null,
-  neutered        boolean not null,
-  weight_kg       float not null,
+CREATE TABLE IF NOT EXISTS owners (
+  id        INT GENERATED ALWAYS AS IDENTITY,
+  full_name VARCHAR(255) NOT NULL,
+  age       INT NOT NULL,
   PRIMARY KEY(id)
 );
 
-ALTER TABLE animals ADD species varchar(40); 
+CREATE TABLE IF NOT EXISTS species (
+  id   INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(40) NOT NULL,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE IF NOT EXISTS animals (
+  id              INT GENERATED ALWAYS AS IDENTITY, 
+  name            VARCHAR(100) NOT NULL,
+  species_id      INT,
+  owner_id        INT,
+  date_of_birth   DATE NOT NULL,
+  escape_attempts INT NOT NULL,
+  neutered        BOOLEAN NOT NULL,
+  weight_kg       FLOAT NOT NULL,
+  PRIMARY KEY(id),
+  CONSTRAINT fk_species
+    FOREIGN KEY(species_id) 
```

Comment:
- [ ] My understanding is that at this point the `animals` table was created in the last milestone. What needs to be done is to modify the table as stated in the requirements. Kindly add the modification queries and unlike the way you have re-created the table.
- Example
![image](https://user-images.githubusercontent.com/81932478/184388679-5d378e3f-2364-4460-9041-1b726cfb3c6b.png)